File: 2485-find-the-pivot-integer/2485-find-the-pivot-integer.py
class Solution:
    def pivotInteger(self, n: int) -> int:
        sum1 = 0
        res = 0
        for i in range(1,n+1):
            sum1 = sum1 + i
            sum2 = 0
            for j in range(i,n+1):
                sum2 = sum2 + j
        
            if sum1 == sum2:
                res = i
                return res
        return -1;
            

File: 2485-find-the-pivot-integer/2485-find-the-pivot-integer.cpp
class Solution {
public:
    int pivotInteger(int n) {
        int sum1 = 0;
        for(int i=1; i<=n; i++){
            sum1 = sum1 + i;
            int sum2 = 0;
            for(int j=i; j<=n; j++){
                sum2 = sum2 + j;
            }
            if(sum1 == sum2){
                return i;
            }
        }
        return -1;
    }
};

